### DISCUSSION TITLE
Thanks to BottlecapDave

### DISCUSSION THREAD

**Discussion by HLFCode** [BottlecapDave/HomeAssistant-OctopusEnergy#1022] [General]
I've been very impressed with the level of positive activity in this really useful repository and I just wanted to say
**Thanks to BottlecapDave**

**Comment by BottlecapDave** [comment]
Thank you for the kind words :)

**Comment by jwt99412** [comment]
Dave, Thanks for spending the time to sort out my issue which must have been an obscure use case :)
